.github/workflows/ci.yaml

```
@@ -0,0 +1,50 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - "**"
+    tags:
+      - "**"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    # Sets the permissions granted to the `GITHUB_TOKEN` for the actions in this job.
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
```

To get arm builds I think you also need setup-qemu

```suggestion
      - name: Set up QEMU
        uses: docker/setup-qemu-action@v3.2.0
        
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v3

```

---

Is that actually still necessary? The `docker-container` build driver setup by the buildx action seems to support multi-platform builds out of the box.
Atleast in my fork the arm images were built without issues: https://github.com/Tarow/immich-public-proxy/pkgs/container/immich-public-proxy

Might be necessary for more "exotic" platforms?

---

I didn't actually check, I'm just used to always using it :sweat_smile: I'll take your word for it :P

---

I'll add it just in case it might be necessary for additional platforms 👍